### PR TITLE
improve plotting to make better use of y range

### PR DESCRIPTION
**Pull request by otteresk**
New calculation of y range to better handle big values with small variations (e.g. no. documents)

### CODE REVIEW

**Comment by otteresk** [comment]
Just wondering if you have questions or comments on this one.

**Comment by Orbiter** [comment]
Hi otteresk, sorry for the delay I have been very busy recently. I will look through your changes as soon as possible.

**Comment by otteresk** [comment]
Hi Orbiter, no problem.

If you would like to see the new version you can check on my node:
http://v22014012007616699.yourvserver.net:18091/Network.html?page=5
http://v22014012007616699.yourvserver.net:18091/NetworkHistory.png?columns=cI&scale=1&maxtime=24&width=660
http://v22014012007616699.yourvserver.net:18091/NetworkHistory.png?columns=cR&scale=1&maxtime=24&width=660

Kind regards, Andreas

**Comment by otteresk** [comment]
Hi Orbiter et al.
may I propose to at least integrate the changes to PerformanceMemory_p.java and ChartPlotter.java - as those are (simple) bug fixes.
Regards, Andreas

**Comment by reger24** [comment]
Hi,
small note fyi, 
the patched Y-scaling has a small side effect on chart with only few data points (min, max equal)

networkhistory-withpatch
networkhistory-withoutpatch

File: source/net/yacy/visualization/ChartPlotter.java [hunk 1]
```diff
@@ -168,7 +168,7 @@ private void drawVerticalScale(final boolean left, final int scale, final int pi
             }
             setColor(colorNaming);
             line(x - 3, y, x + 3, y, 100);
-            s1 = (s >= 1000000 && s % 10000 == 0) ? Integer.toString(s / 1000000) + "M" : (s >= 1000 && s % 1000 == 0) ? Integer.toString(s / 1000) + "K" : Integer.toString(s);
+            s1 = (s >= 1000000 && s % 1000000 == 0) ? Integer.toString(s / 1000000) + "M" : (s >= 1000 && s % 1000 == 0) ? Integer.toString(s / 1000) + "K" : Integer.toString(s);
```
Comment (reger24) [inline source/net/yacy/visualization/ChartPlotter.java]: fyi: this bugfix will be applied for sure

**Comment by otteresk** [comment]
Thanks for the test and feedback, Burkhard.
I will look into this and come back.

**Comment by otteresk** [comment]
Should be fixed now.

**Comment by otteresk** [comment]
I really have to get used to git(hub) :-)
Now NetworkHistory.java should be able to handle all kind of cases.
Thanks for testing and sorry for any inconveniences.

**Comment by otteresk** [comment]
Hi Burkhard, any chance you find time for another test?

**Comment by reger24** [comment]
Hi @otteresk, I believe the mentioned issue you've resolved.
but I still would like to leave it up to @Orbiter to accept or reject you pull request.

**Pull request merged**